static/app/views/settings/account/mergeAccounts.tsx

```diff
@@ -0,0 +1,343 @@
+  onSelect: (newUserId: string) => void;
+  users: UserWithOrganizations[];
+};
+
+type UserRowProps = {
+  id: string;
+  lastSeen: string;
+  name: string;
+  onSelect: (newUserId: string) => void;
+  organizations: string;
+};
+
+type UserProps = {
+  onSelect: (newUserId: string) => void;
+  users: UserWithOrganizations[];
+};
+
+function Users({users, onSelect}: UserProps) {
+  const isPrimaryUser = useContext(UserContext);
+  if (isPrimaryUser) {
+    return (
+      <Panel>
+        <UserPanelHeader>
+          <div>{t('Name')}</div>
+          <div>{t('Last Active')}</div>
+          <div>{t('Organizations')}</div>
+        </UserPanelHeader>
+        <PanelBody>
+          {users.map(userObj => (
+            <UserRow
+              onSelect={onSelect}
+              key={userObj.id}
+              id={userObj.id}
+              name={userObj.name}
+              lastSeen={userObj.lastActive}
+              organizations={userObj.organizations.join(', ')}
+            />
+          ))}
+        </PanelBody>
+      </Panel>
+    );
+  }
+  return (
+    <Panel>
+      <UserPanelHeader>
+        <div>{t('Name')}</div>
+        <div>{t('Last Active')}</div>
+        <div>{t('Organizations')}</div>
+        <div>{t('Merge')}</div>
+      </UserPanelHeader>
+      <PanelBody>
+        {users.map(userObj => (
+          <UserRow
+            onSelect={onSelect}
+            key={userObj.id}
+            id={userObj.id}
+            name={userObj.name}
+            lastSeen={userObj.lastActive}
+            organizations={userObj.organizations.join(', ')}
+          />
+        ))}
+      </PanelBody>
+    </Panel>
+  );
+}
+
+function UserRow({id, name, lastSeen, organizations, onSelect}: UserRowProps) {
+  const isPrimaryUser = useContext(UserContext);
+
+  return (
+    <UserPanelItem>
+      <Name>{name}</Name>
+      {isPrimaryUser ? (
+        'Currently active'
+      ) : lastSeen === '' ? (
+        'Never'
+      ) : (
+        <div>
+          <StyledTimeSince date={lastSeen} />
+        </div>
+      )}
+      <div> {organizations} </div>
+      {isPrimaryUser ? null : (
+        <div>
+          <input
+            type="checkbox"
+            name="user"
+            value={name}
+            onChange={() => onSelect(id)}
+            style={{margin: 5}}
+          />
+        </div>
+      )}
+    </UserPanelItem>
+  );
+}
+
+const tableLayout = `
+  display: grid;
+  grid-template-columns: auto 140px 140px 60px;
+  gap ${space(1)};
+  align-items: center;
+`;
+
+const UserPanelItem = styled(PanelItem)`
+  ${tableLayout};
+`;
+
+const Name = styled('div')`
+  margin-bottom: ${space(0.5)};
+  font-weight: ${p => p.theme.fontWeight.bold};
+`;
+
+const StyledTimeSince = styled(TimeSince)`
+  font-size: ${p => p.theme.fontSize.sm};
+`;
+
+const UserPanelHeader = styled(PanelHeader)`
+  ${tableLayout}
+  justify-content: initial;
+`;
+
+const StyledListItem = styled(ListItem)`
+  margin-bottom: ${space(0.5)};
```

Comment:
instead of adding margin to every component can we use grid or flex to add a gap between each section?